Cython/Utility/ModuleSetupCode.c

```diff
@@ -169,7 +172,7 @@
   #undef CYTHON_USE_EXC_INFO_STACK
   #define CYTHON_USE_EXC_INFO_STACK 0
   #ifndef CYTHON_UPDATE_DESCRIPTOR_DOC
-    #define CYTHON_UPDATE_DESCRIPTOR_DOC (PYPY_VERSION_HEX >= 0x07030900)
+    #define CYTHON_UPDATE_DESCRIPTOR_DOC (PYPY_VERSION_NUM >= 0x07030900)
```

Comment:
This is probably worth picking to 0.29.x. I'm inclined not to pick the rest on the basis that we're largely trying not to change it.

Reply:
OK. I can drop the last commit and just leave the typo in this PR.

Reply:
Wasn't asking you to do anything with this PR. Was just noting that it's worth making a 0.29.x PR with just that change

Reply:
No problem, let's consider this the 0.29.x PR :)